sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsLifecycleSamples.cs

```diff
@@ -263,12 +263,8 @@ public async Task CreateAllTwinsAsync()
             {
                 try
                 {
-                    #region Snippet:DigitalTwinsSampleCreateTwin
```

Comment:
This one liner was an awful example, so I chose to show how one would construct a digitaltwin using the BasicDigitalTwin type.

I plan to add another using a custom POCO, and maybe one using JsonElement.